yarpp-template-unwind.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/*
+YARPP Template: SiteOrigin Unwind
+*/ ?>
+<h2 class="related-posts heading-strike"><?php esc_html_e( 'You may also like', 'siteorigin_unwind' ); ?></h2>
```

Comment:
I felt there was something wrong. You must have accidently used `siteorigin_unwind` as the theme slug while generating the theme from underscores.me. The translation domain is `siteorigin_unwind` instead of `siteorigin-unwind` across all templates.